optuna/integration/_lightgbm_tuner/sklearn.py

```diff
@@ -1,49 +1,854 @@
+            ``{class_label: weight}``. This parameter is used only for
+            multi-class classification task. For binary classification task you
+            may use ``is_unbalance`` or ``scale_pos_weight`` parameters. The
+            'balanced' mode uses the values of y to automatically adjust
+            weights inversely proportional to class frequencies in the input
+            data as ``n_samples  (n_classes * np.bincount(y))``. If None, all
+            classes are supposed to have weight one. Note, that these weights
+            will be multiplied with ``sample_weight`` if ``sample_weight`` is
+            specified.
+
+        min_split_gain:
+            Minimum loss reduction required to make a further partition on a
+            leaf node of the tree.
+
+        min_child_weight:
+            Minimum sum of instance weight (hessian) needed in a child (leaf).
+
+        min_child_samples:
+            Minimum number of data needed in a child (leaf).
+
+        subsample:
+            Subsample ratio of the training instance.
+
+        subsample_freq:
+            Frequence of subsample. <=0 means no enable.
+
+        colsample_bytree:
+            Subsample ratio of columns when constructing each tree.
+
+        reg_alpha:
+            L1 regularization term on weights.
+
+        reg_lambda:
+            L2 regularization term on weights.
+
+        random_state:
+            Seed of the pseudo random number generator. If int, this is the
+            seed used by the random number generator. If
+            ``numpy.random.RandomState`` object, this is the random number
+            generator. If None, the global random state from ``numpy.random``
+            is used.
+
+        n_jobs:
+            Number of parallel jobs. -1 means using all processors.
+
+        silent:
+            If true, print messages while running boosting.
+
+        importance_type:
+            Type of feature importances. If 'split', result contains numbers of
+            times the feature is used in a model. If 'gain', result contains
+            total gains of splits which use the feature.
+
+        study:
+            Study corresponds to the optimization task. If None, a new study is
+            created.
+
+        timeout:
+            Time limit in seconds for the search of appropriate models. If
+            None, the study is executed without time limitation. If
+            ``n_trials`` is also set to None, the study continues to create
+            trials until it receives a termination signal such as Ctrl+C or
+            SIGTERM. This trades off runtime vs quality of the solution.
+
+        model_dir:
+            Directory for storing the files generated during training.
+
+        **kwargs:
+            Other parameters for the model. See
+            http:/lightgbm.readthedocs.io/en/latest/Parameters.html for more
+            parameters. Note, that **kwargs is not supported in sklearn, so it
```

Comment:
Please escape `**kwargs` to avoid syntax error.

```suggestion
            parameters. Note, that ``**kwargs`` is not supported in sklearn, so it
```

Reply:
Fixed in 9351d0f.

Reply:
Thanks!